server/controller/survey.go

```diff
@@ -16,19 +17,40 @@ var getSurvey = &Endpoint{
 	RequiresAuth: true,
 }
 
+// Get survey by meetingID or by surveyID and surveyVersion
 func executeGetSurvey(w http.ResponseWriter, r *http.Request) error {
```

Comment:
Maybe we don't need to return error from controller. We can send the error as a response.

Reply:
This return value is only used for logging purposes. We do send the error as a response.